src/style.css

```
@@ -0,0 +1,83 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.search {
+  font-family: Avenir, sans-serif;
+  color: #3d4e61;
+  box-sizing: border-box;
+  display: flex;
+
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+  margin-top: 20px;
+  margin-bottom: 20px;
+  height: 70px;
+}
+
+.search:hover {
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search:focus-within {
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  font-weight: bold;
+}
+
+.searchField {
+  color: inherit;
+  font-family: inherit;
+  font-weight: inherit;
+  border: 0;
+  outline: 0;
+  background-color: transparent;
+}
+
+.searchIcon--large {
+  width: 19px;
+  height: 19px;
+  margin: 24px 18px 26px 25px;
+}
+
+.searchField--large {
```

```suggestion
.searchField {
```
2 point in checklist